change/@fluentui-react-infolabel-3073ec02-8c14-4fef-976d-39ee32603aa0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33057&commitId=621e85e7ec46543820aed139adccfb489078c664&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|3279| Changed|
|Drawer.overlay drawer full.chromium.png|3301| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>